corehq/form_processor/reprocess.py

```diff
@@ -39,9 +39,11 @@ def reprocess_unfinished_stub(stub, save=True):
     try:
         form = FormAccessors(stub.domain).get_form(form_id)
     except XFormNotFound:
-        # form doesn't exist which means the failure probably happend during saving so
-        # let mobile handle re-submitting it
-        logger.error('Form not found: %s', form_id)
+        if stub.saved:
```

Comment:
Why only log saved forms?

Reply:
if there was an error saving then we'd expect the error to be raised.